configs/notion-formula.json

```
@@ -1,7 +1,7 @@
 {
-  "index_name": "notion-formula",
+  "index_name": "nishan",
```

Hi,

If you'd like to change the index name, please send us an email directly, as it requires a few changes. 

In the meantime, you should revert it. 

```suggestion
  "index_name": "notion-formula",
```